subsys/bluetooth/controller/include/ll.h

```diff
@@ -7,6 +7,12 @@
 
 #define LL_VERSION_NUMBER BT_HCI_VERSION_5_1
 
+enum bt_hci_vs_handle_type {
```

Comment:
Why redefine the handle type again here when the values are in hci_vs.h ? Drop this, and use u8_t in ll_tx_pwr_lvl_set/get.

Reply:
The main reason behind this was, @joerchan please correct me if I am wrong in any way, to allow others to use this pattern in the LL split for other implementations, whereas the definitions in the hci_vs.h particular to this implementation where provide for the HCI interface of the set/get tx power level commands.

Reply:
I think I wanted the enum to be independant of TX power, so that it could be re-used for similar features that can be configured for all LL roles. Redefining the enum is probably not wise, so should include `hci_vs.h` instead and use the definition from there. If u8_t is desired as the argument type then it is probably best to follow that advice. But need to include the `hci_vs.h` header for the source file where the switch on the enum values are.

Reply:
Ok, @joerchan, I understand. To resolve both of your request and @cvinayak's, I suggest the following:

- remove `enum bt_hci_vs_handle_type` completely and use `u8_t` instead
- use macro define values from `hci_vs.h` and thus indeed include  `hci_vs.h` where ever there are checks on the type of the handle
- rename macro define names to make them more generic ? and thus not tie them to the power control and encourage the pattern for other purposes (here the idea is to change the name from `BT_HCI_VS_LL_TX_POWER_CONTROL_*` to `BT_HCI_VS_LL_HANDLE_TYPE_*` to make the enum independent of the TX power)

Would these be then okay for both ?

Reply:
Fixed as discussed - [`hci_vs.h`](https://github.com/zephyrproject-rtos/zephyr/pull/19779/commits/856709c837c8feb553e54c4b84fc44174136897f#diff-d32c7d39d131dcc56d85c8b8b7d5b328R133) and [`ll.h`](https://github.com/zephyrproject-rtos/zephyr/pull/19779/commits/856709c837c8feb553e54c4b84fc44174136897f#diff-65c5dce42a06bd585801995fc7a03ad0R84)